### ISSUE TITLE
Incompatible with custom post types

### ISSUE THREAD

**Issue by elbnetz** [luckyduck/Simple-Multisite-Sitemaps#1]
Hi There,

Attention: This plugin has made our multisite network incompatible with Custom Post Types. Unfortunately, the problem can't be fixed by deinstalling the plugin. The plugin may be doing something with rewrite rules at the wrong time during initialization.

We have a NGNX server. We do not know if that plays a role.

We have contacted the developer and wait for a reply and assistance in correcting the error.

**Comment by luckyduck** [comment]
I use it on my own and have no problems whatsoever. If you take a look at the code, there is no way to interfere with your custom post type.

**Comment by luckyduck** [comment]
In which way does it generate problems? What does this mean:

[...]
multisite network incompatible with Custom Post Types
[...]

**Comment by elbnetz** [comment]
Can we talk? I tried to reach you by phone and sent you all information we have so far.

**Comment by luckyduck** [comment]
Yeah, I saw that. I'm currently in the office. Tomorrow is a holiday over here (NRW). Maybe on friday?

**Comment by elbnetz** [comment]
Could we discuss it today? If not, Friday will do. Please call as it is impossible to get you on the phone.

**Comment by luckyduck** [comment]
Problem is, I'm in the office and when I get home, my little son will be waiting for me. He's going to a vacation with his grandpa/grandma tomorrow morning. I can't give you a call before tomorrow afternoon. :/

**Comment by elbnetz** [comment]
Hi, we would appreciate your call as we have serious problems with this plugin. It should be in your interest to solve the issue as it may cause problems elsewhere as well.

Thanks.

**Comment by elbnetz** [comment]
Hey, why are you not calling back?
Are you not interested solving the problem?

**Comment by luckyduck** [comment]
Well, I havn't had the chance, yet. After all, this project is free as in free software and free as in free beer. I develop it in my spare time and have no problems what so ever. I had no free time, yet.

**Comment by elbnetz** [comment]
I do understand that. But it seems that this plugin has a critical malfunction which is irreversible after installation. Don't you think that should be fixed?

**Comment by luckyduck** [comment]
Should be fixed, but it has no critical malfunction. As I said, I used this plugin on my own sites. Works fine. All it does is to show a sitemap. It only interacts with WordPress as it flushes rewrite rules.

Mit freundlichen Grüßen,
Best Regards

Jan Brinkmann

luckyduck networks
Alte-Rothe-Str. 40a
D-33189 Schlangen

Tel: +49 5252 2692517 - 0
Fax: +49 5252 2692517 - 9

Web - http://the-luckyduck.de
Twitter - @j_brinkmann - Follow me!

**Comment by elbnetz** [comment]
We can replicate the problem. It is your plugin that causes that described malfunction, which is critical to us and will be for others. 

As you can see in the document we sent you it even deletes custom post type entries. Plus it is not possible to reactivate custom post types after deinstallation of your plugin by re install theme/plugin. 

We expect this due to your rewrite rules which will stay after installation but are no experts in that field.

**Comment by elbnetz** [comment]
Hi There,

When can we expect your kind support? 
Or is it something you don't want to solve?

Thanks and have a nice day.

**Comment by elbnetz** [comment]
Flushing the rewrite rules is an expensive operation, there are tutorials and examples that suggest executing it on the 'init' hook. This is bad practice.

Thanks for not helping.
